### DISCUSSION TITLE
Create snippets for cmdline

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [L3MON4D3/LuaSnip#1080] [Q&A]
Is there a way to create snippets just for command line ?

**Comment by L3MON4D3** [comment]
Oof, no, I don't think that's possible

**Reply by L3MON4D3** [reply]
(as in it's not possible now, if I haven't missed something completely major, and implementing it seems pretty infeasible)

**Reply by LamprosPitsillos** [reply]
Really sad to hear... thanks!
